index.html

```diff
@@ -251,50 +291,42 @@ <h2>Professional Art Printing Data</h2>
         >
           See Project
         </button>
-      </article>
-    </section>
+      </div>
+      </div>
+    </div>
+    
     <!-- Start About me-->
-    <section class="about-me">
-      <article>
+    <div class="about-me">
+      <div class="backgroundabout-me">
       <div class="first">
         <h2>About me</h2>
```

Comment:
- [ ] [OPTIONAL] I think your portfolio has a few alignment issues that have not been addressed before and I will mark these as [OPTIONAL] but I strongly suggest working on these before moving to the next milestone :pushpin: :raised_hands:

  About section: The "About me" title should be aligned with the paragraph :pushpin:
  ![image](https://user-images.githubusercontent.com/81522435/140470763-ad91a4e1-b74f-4f89-a507-ff6b532f0f84.png)

  Knowledge section: I should be centered according to the Figma design :wink:
  ![image](https://user-images.githubusercontent.com/81522435/140470806-44d47ab8-20e5-4623-a06c-e6afbbe82bda.png)

  Contact form section: This form should also be centered in order to match the Figma design :hugs:
  ![image](https://user-images.githubusercontent.com/81522435/140470842-4a792f08-ecc5-4e22-a0fd-aa470bb413f0.png)